src/components/FacebookButton/FacebookButton.jsx

```diff
@@ -0,0 +1,27 @@
+import { auth } from "@/util/firebase";
+import { FacebookAuthProvider, signInWithPopup } from "firebase/auth";
+import { BsFacebook } from "react-icons/bs";
+
+export default function FacebookButton({ children }) {
+    function signInWithFacebook() {
+        const provider = new FacebookAuthProvider();
+        signInWithPopup(auth, provider)
+            .then((result) => {
+                console.log(result);
```

Comment:
Don't forget to get rid of the console logs later.

Reply:
thank you for your feedback i will git rid of them.